styles.css

```diff
@@ -85,14 +155,345 @@ nav {
   justify-content: center;
   align-items: center;
   padding: 0;
-  margin-top: 70px;
+  margin-top: 10%;
   list-style-type: none;
 }
 
+.scroll {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.scroll div {
+  width: 40px;
+  height: 40px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit);
+  margin-top: 62px;
+  justify-content: center;
+  align-items: center;
+  row-gap: 25px;
+}
+
+.grid-item {
+  display: grid;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin: 0 auto;
+  width: 327px;
+  height: 510px;
+}
+
+.grid-item-top {
+  background: #ebf0ee;
+  height: 250px;
+}
+
+.grid-item-bottom {
+  background: white;
+  height: 260px;
+}
```

Comment:
For the works section design of your project, I'd kindly suggest you implement the following changes :

-  please reduce the height a bit to be similar to what we have for each project are your project cards look longer than what is expected for a better view 😄 

- please add a border-radius property for your project cards, so it looks similar to the Figma template given

- Please use the right font-color for the `multi post stories` which is `background: #3A4A42` 

Figma design :
![image](https://github.com/NgachaIan/Personal_Portfolio/assets/20537712/6e4e525e-ca5c-45a5-8af9-6f4af122f0dc)

Your design :
![image](https://github.com/NgachaIan/Personal_Portfolio/assets/20537712/6d673c8e-df5e-41a3-b3e4-f5c59e5cc3fc)